mesonbuild/backend/xcodebackend.py

```diff
@@ -53,7 +53,8 @@
                 }
 BOOL2XCODEBOOL = {True: 'YES', False: 'NO'}
 LINKABLE_EXTENSIONS = {'.o', '.a', '.obj', '.so', '.dylib'}
-XCODEVERSIONS = {'1400': ('Xcode 14.0', 56),
+XCODEVERSIONS = {'1500': ('Xcode 15.0', 60),
+                 '1400': ('Xcode 14.0', 56),
```

Comment:
Any particular reason this gets its own commit?

Reply:
This was before Xcode 15 was released. I will move this into an existing commit.